<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <class>QmitkDicomTractogramTagEditorViewControls</class>
 <widget class="QWidget" name="QmitkDicomTractogramTagEditorViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <property name="styleSheet">
   <string>QCommandLinkButton {
  font-weight: lighter;
}

QCommandLinkButton:disabled {
  border: none;
  font-weight: lighter;
}

QToolBox::tab {
  font-weight: bold;
}

QToolBox::tab:hover {
  font-weight: bold;
}

QToolBox::tab:selected {
  font-weight: bold;
}

QGroupBox {
  background-color: transparent;
}

QGroupBox, QGroupBox:disabled {
  background-color: transparent;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="toolTip">
         <string></string>
        </property>
        <property name="text">
         <string>Tractogram:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_TractogramBox">
        <property name="toolTip">
         <string>Seed points are only placed inside the mask image. If no seed mask is selected, the whole image is seeded.</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tractography DICOM Tags</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTableWidget" name="m_TagTable">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QCommandLinkButton" name="m_CopyPropsButton">
     <property name="toolTip">
      <string>Copies the DICOM properties of the tractogram selected in this view to the one selected in the datamanager.</string>
     </property>
     <property name="text">
      <string>Copy Properties From Image</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"></layoutdefault>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBoxWithSelectNone</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBoxWithSelectNone.h</header>
  </customwidget>
 </customwidgets>
 <resources></resources>
 <connections></connections>
</ui>